insect_tools.py

```diff
@@ -1981,17 +1981,17 @@ def visualize_wing_shape_file(fname, ax=None, fig=None, savePNG=True):
         
         plt.pcolormesh(Y.T, X.T, mask, rasterized=True, cmap='gray_r')    
         
-    x0 = wabbit_tools.get_ini_parameter(fname, "Wing", "x0w", float)
-    y0 = wabbit_tools.get_ini_parameter(fname, "Wing", "y0w", float)
+    x0 = wabbit_ini_tools.get_ini_parameter(fname, "Wing", "x0w", float)
+    y0 = wabbit_ini_tools.get_ini_parameter(fname, "Wing", "y0w", float)
     
     
     #--------------------------------------------------------------------------
     # planform (contour)
     #--------------------------------------------------------------------------
     if wtype == "fourier":
-        a0 = wabbit_tools.get_ini_parameter(fname, "Wing", "a0_wings", float)
-        ai = wabbit_tools.get_ini_parameter(fname, "Wing", "ai_wings", float, vector=True)
-        bi = wabbit_tools.get_ini_parameter(fname, "Wing", "bi_wings", float, vector=True)
+        a0 = wabbit_ini_tools.get_ini_parameter(fname, "Wing", "a0_wings", float)
```

Comment:
moving those functions from `wabbit_tools` to `wabbit_ini_tools` is incredibly painful, as it affects thousands of scripts.

If we do it, I insist on a code-neutral name: `ini_tools` or `inifile_tools` because it can be used for other codes as well.

Reply:
Please include the functions in `wabbit_tools` just as a wrapper to `inifile_tools`, and add a deprecation warning:
`Using wabbit_tools.get_ini_parameter is deprecated. Please use inifile_tools.get_ini_parameter instead`